packages/basic/src/typography.scss

```
@@ -0,0 +1,383 @@
+$line-height: 24px;
+
+$h1-font-size: 40px;
```

Stylelint (17.4.0)

[error] Unexpected empty line before $-variable (scss/dollar-variable-empty-line-before)

(scss/dollar-variable-empty-line-before)